code_search/src/code_search/dataflow/cli/arguments.py

```diff
@@ -42,6 +42,14 @@ def add_parser_arguments(parser):
                       help=('If specified read the entire GitHub dataset '
                             'specified as PROJECT:DATASET.TABLE. If not '
                             'specified we run a query to filter the data.'))
+  additional_args_parser.add_argument(
+                      '--read_github_dataset_for_function_embedding', action='store_true',
```

Comment:
Do we need the argument --read_github_dataset_for_function_embedding?

Could we infer based on other arguments? e.g. if we read the input from the token_pairs table we could have an argument for the token_pairs table. If that isn't set then we could read from the BigQuery GitHub dataset.

Reply:
done

Reply:
It looks like this parameter is still here?